.github/workflows/production.yml

```diff
@@ -1,12 +1,14 @@
-name: Provision EC2 Instance with Terraform
+name: Provision EC2 Instance in production with Terraform
 
-on: [workflow_dispatch]
+on:
+  push:
+    branches:
+      - main
```

Comment:
Can we trigger pipeline manually?

Reply:
Only manual please. We need to manage the deployment manually and don't want an automatic process to trigger it for us.

Reply:
Changed